File: iteration1/CSE3063F21P1_GRP16/src/Models/CourseSection.java
package Models;

import java.util.ArrayList;

public class CourseSection {
    private ArrayList<Student> allstudent = new ArrayList<Student>(); // Array list that includes all the students
                                                                      // registered to a specific course
    // private ArrayList<CoursesError> errors = new ArrayList<CoursesError>();
    // //Array list that includes errors while registering to a course

    // Counstructor
    public CourseSection(ArrayList<Student> allstudent /* ArrayList<CourseError> errors */) {
        this.allstudent = allstudent;
        // this.errors = errors;
    }

    // Setters and Getters
    public ArrayList<Student> getAllstudent() {
        return allstudent;
    }

    public void setAllstudent(ArrayList<Student> allstudent) {
        this.allstudent = allstudent;
    }

    /*
     * public ArrayList<CoursesError> getErrors() {
     * return errors;
     * }
     * 
     * //public void setErrors(ArrayList<CoursesError> errors) {
     * this.errors = errors;
     * }
     */

    ArrayList<Course> selectedCourses = new ArrayList<Course>();

    public ArrayList<Course> getSelectedCourses() {
        return selectedCourses;
    }

    public void setSelectedCourses(ArrayList<Course> selectedCourses) {
        this.selectedCourses = selectedCourses;
    }

}

File: iteration3/CSE3063F21P1_GRP16/models/CourseSection.py
class CourseSection:
    def __init__(self, allstudent):
        self.__allstudent = allstudent
        self.__selectedCourses = []

    def getAllstudent(self):
        return self.__allstudent

    def setAllstudent(self, allstudent):
        self.__allstudent = allstudent

    def getSelectedCourses(self):
        return self.__selectedCourses

    def setSelectedCourses(self, selectedCourses):
        self.__selectedCourses = selectedCourses
